<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*{
	font-family:&quot;微软雅黑&quot;;
	outline:0px;
}

#MainWindow,#centralWidget
{
	background-color:#ececec;
}

#scrollArea,#scrollAreaWidgetContents
{
	background-color:#ececec;
}

#toolButton_switchinfo
{
	border-style:none;
	min-height:80px;
	max-height:80px;
	background-color:transparent;
}

#widget_machineState
{
	border-style:none;
	min-height:80px;
	max-height:80px;
}

#widget_userinfo
{
	border-style:none;
	min-width:100px;
	max-width:100px;
}

#label_username
{
	border-style:none;
	color:blue;
}

#label_userlevel
{
	border-style:none;
	color:orange;
}

QLineEdit
{
    border-style: none;
    color: #7F848D;
	border: 1px solid #cecece;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #D0D0D0, stop:1 #D5D5D5);
	border-radius: 4px;
	min-height:24px;
	min-width:64px;
}

QLineEdit:hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #DADADA, stop:1 #DFDFDF);
}

QProgressBar{
	min-height:10px;
	border-radius:2px;
	text-align:center;
	background-color:#dedede;
}

#pushButton_status
{
	border-radius:2px;
	background-color:#ececec;
}

#pushButton_status:hover
{
	border-radius:2px;
	background-color:#dedede;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_Connect" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>默认设置</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="3" column="0">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>协议标识：</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_DefaultProtocol">
                <property name="text">
                 <string>0000</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_DefaultWriteCode">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>读功能码：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>写功能码：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>传输标志：</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_DefaultReadCode">
                <property name="text">
                 <string>03</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_DefaultTransmission">
                <property name="text">
                 <string>0000</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <widget class="QPushButton" name="pushButton_UpdataDefaultSet">
                <property name="text">
                 <string>修改默认设置</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>IP地址：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_IP">
               <property name="text">
                <string>192.168.88.100</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>端口号：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_Port">
               <property name="text">
                <string>502</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_Connect">
               <property name="text">
                <string>连接</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_DisConnect">
               <property name="text">
                <string>断开连接</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QTextEdit" name="textEdit_ConnectState"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_Read" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>读值</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_Protocol">
                  <property name="text">
                   <string>0000</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>数据起始地址：</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_Length">
                  <property name="text">
                   <string>0001</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>寄存器数量(word)：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_StartAddress">
                  <property name="text">
                   <string>0001</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_PDU">
                  <property name="text">
                   <string>0006</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>传输标志：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>PDU长度：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_FunctionCode">
                  <property name="text">
                   <string>03</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>功能码：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_addressCode">
                  <property name="text">
                   <string>01</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_Read_Transmission">
                  <property name="text">
                   <string>0000</string>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>地址码：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>协议标识：</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>数据类型：</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QComboBox" name="comboBox_Read_DataFormat">
                  <item>
                   <property name="text">
                    <string>Float（2word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Int（1word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>DInt（2word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>str</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>bool（1word）</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QTextEdit" name="textEdit_Read"/>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="pushButton_Read">
                <property name="text">
                 <string>读取</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_Write" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>写值</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_Transmission">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_PDU">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>地址码：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>数据起始地址：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>功能码：</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>写word的数量：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_FunctionCode">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_WordNum">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_StartAddress">
                  <property name="text">
                   <string>01</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_addressCode">
                  <property name="text">
                   <string>01</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>传输标志：</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>写字节的数量：</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_byteNum">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_Data"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>协议标识：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_Write_Protocol">
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>PDU长度：</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>数据值：</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QComboBox" name="comboBox_Write_DataFormat">
                  <item>
                   <property name="text">
                    <string>Float（4byte-2word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Int（2byte-1word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>DInt（4byte-2word）</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>str</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>bool（2byte-1word）</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>数据类型:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QTextEdit" name="textEdit_Write"/>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="pushButton_Write">
                <property name="text">
                 <string>写入</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
